### ISSUE TITLE
Standards Page

### ISSUE THREAD

**Issue by ben-norton** [tdwg/website#478]
A couple comments regarding the Standards page.
1. The opening paragraph could be informative and explicit by defining the 'status' types and categories. Here's my working paragraph.
All TDWG standards have a status and assigned category. "Current" standards official TDWG data standards actively maintained through a designated maintenance group. For more information about official standards and maintenance groups, please visit our ** page. The "Draft" status is assigned to standards currently under review, a step required before official ratification. For more information about the ratification process, please see ***. (Additional status types listed on the page).
Our standards are also available as a collection on FAIRsharing.org, where you can discover how and where our standards are used across the bioinformatics community.
2. In regards to FAIRsharing, I understand the special regard many of us have for FAIR, but I'm sure there are several locations where people can see how and where TDWG standards are used. Should we link to a more comprehensive table? Is the intent to provide special consideration for FAIRsharing? For the uninitiated, it looks out of place.
3. The categories are confusing. At the top, two status types are described, Current and Draft. Then the standards are listed under Current, 2005, and Prior. What is a '2005' standard? Is that a status? What is a 'Prior' standard? 
I think we would open the page with a short overview, followed by a short description of each standard status, then group them accordingly below. For example, a sentence is added explaining what a 'prior' standard is (may not be 10)% accurate)
'Prior' standards have undergone the official ratification process, but are no longer actively maintained.

**Comment by baskaufs** [comment]
The chief distinction between current and prior standards is that current standards have gone through the current TDWG Process (task group, review manager, public comment, etc.) and whose documents conform to the Standards Documentation Specification. Prior standards have not. 

I hope that we can soon just make the Current (2005) category go away. ABCD and TCS have or soon will have maintenance groups that are working on making "modern" versions of them that conform to the SDS. At that point they will move out of the Current (2005) category. As far as I know, there isn't any effort being made to move the other two into the regular "Current Standard" category. Unless a champion emerges who's willing to do the work to make them conform to the SDS, I will probably propose that they just get moved into the prior standards category and eliminate the Current (2005) category.

Kit Lewers and I (in the name of the TAG) are doing a comprehensive review of all of the standards and I expect that there will be recommendations about this (and about retiring some unused standards) at the end of that process. So for now I just wouldn't worry about this in the context of the website and let it sort itself out over the next year.

**Comment by ben-norton** [comment]
@baskaufs 
In that case, is there a better name for them? Current is temporal, but they are  grouped by status and process. Would "Official" or something similar be better?
I always found current to be inconclusive. In other words, why is a standard classified as 'Current'?

**Comment by baskaufs** [comment]
> The "Draft" status is assigned to standards currently under review, a step required before official ratification. 

I think I've previously expressed an opinion that draft standards shouldn't be listed on the standards page. The only reason that's been the case in the past is because we've had standards that have been "stuck" there for years or decades. I think we've finally gotten rid of all of those (either by finishing them or by removing them because they will never be finished), and I don't see any reason to ever list any draft standards on the standards page again. They only need to be exposed as drafts during the public comment period, which with the 30 day comment period and final editing should not last more than a few months. And during that time, those drafts can live on the task group's GitHub repo.

The only reason drafts ever ended up on this page was because in the beginning, the standards were listed automatically by the Typo3 system that was used to generate everything and to manage reviews. At this point, listing draft standards should be considered a bug, not a feature.

**Comment by baskaufs** [comment]
> In that case, is there a better name for them? Current is temporal, but they are grouped by status and process. Would "Official" or something similar be better?
> I always found current to be inconclusive. In other words, why is a standard classified as 'Current'?

I agree that "current" isn't great. I'm open to suggestion. Ultimately, I think there should be three categories: standards that conform to current documentation and management practices, old ones that don't, and retired standards. We don't have any retired standards, but probably should once we figure out which ones aren't used by anybody anymore. The Current (2005) category should go away sometime in the near future.

**Comment by ben-norton** [comment]
@baskaufs 
Suggested status terms (needs work, but the criteria is all there). 

Legacy standards: These standards are recognized as official standards with active support from TDWG. Adherence to TDWG policies is evaluated during formal ratification. In contrast to **, the legacy statis is assigned to standards ratified prior to **, when the current policies were put in place. Therefore, they do not conform to current policies, but remain fully support from TDWG. 

Deprecated Standards
Standards are assigned the deprecation status when they are no longer supported by TDWG. This is commonly referred to as 'end of life' status. Deprecated standards should not be used for data standardization activities. They are posted for reference, historical record,  transparency, and to maintain proper accreditation for past efforts. 

Official Standards
Official status is assigned to standards that 1) conform to current policies, 2) tecieved for ratification, 3) provided active support from the TDWG organization.


I'm not confident in the term Official, but I think legacy and deprecated are on point (they also connect standards to technology by reusing popular terminology in computing). 
Deprecated is functionally equivalent to github repos that are set to read-only.

**Comment by baskaufs** [comment]
I agree that "Legacy" makes sense as a category. 

"Official" doesn't seem like the right term to me. Anything with the TDWG stamp of approval past or present would be official. "Current" still seems OK to me, as the criteria for that category is that they've gone through the current process of review and are formatted to current specifications.

Not sure about your definition of "Deprecated". @DavidFichtmueller pointed out that there is a difference between standards that would be no longer recommended for use (because they aren't being used by anyone) and standards that are not being maintained by TDWG, but may not really need maintenance because they are stable. There is a third category of works that were adopted as standards by TDWG in the past, but which are still useful and being maintained by some other group. Index Herbariorum comes to mind. It seems like all three of these categories are distinct but could all be said to be "no longer supported by TDWG". 

I think it might make sense to review this terminology after the TAG standards review is finished. That might make it clearer what categories are actually needed at the present. Also, I'm not sure if there is any official "rules" about these names. This would be important to know if we are talking about changing them. @stanblum might know.  I've just seen them on the website unchanged for as long as I've been looking at it. The other categories (Technical specification, Best practice, etc.) come straight from IETF if I'm remembering correctly.

**Comment by stanblum** [comment]
I'm OK with both

   - Current -- (as you defined), and
   - Legacy -- not compliant with current documentation standards; possibly
   maintained and used.  (ABCD, EFG, GGBN would fall into this category, but
   the users might not be happy about that label. They've had 6+ years to
   revise things, but we should be cognizant of their situation:  they haven't
   been motivated to update the format because their systems use the XML
   schema and work.  ABCD *has* done incremental updates, without doing the
   format change.  Social considerations are very important in TDWG, as we try
   to keep everyone in the same forum.

I agree that Deprecated is somewhat special; meaning that we don't
recommend it anymore, which probably means there is a better way to achieve
the same or better results.

We copied the Draft status from IETF, where testing of Draft standards is
crucial to their acceptance.  "Draft" signals that the authors believe the
specification meets the objectives, is workable, and implementers/testers
are welcome, with the caveat that the spec might change to accommodate
feedback (and without the usual change process).  Ideally, Drafts in our
process would be very short lived, but that hasn't always been the case.
We impose time limits on public review, and the underlying assumption is
that without objection the standard will be ratified.  Expert review is
where we expect the most meaningful feedback, but those reviews can take a
long time if people drop the ball, or if the Draft is large and complicated.

Best,

-Stan
